smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -122,6 +122,22 @@ contract MixinKeys is
     _;
   }
 
+  /**
+   * Get a key owned by a specific address
+   * @return The key owned by `_keysOwner`
+  */
+  function getKeysByOwner(
+    address _keysOwner
+  ) 
+    public
+    view
+    returns (
+      Key memory 
```

Comment:
This should return an array though :)

Reply:
Yes sure. First I need to take out the pointer-related logic so we can use getters/setters for the mapping. 

I am removing the plural for now, will reintroduce later